src/settings.ts

```diff
@@ -0,0 +1,42 @@
+export enum MigrationMode {
+  ALL,
+  ONLY_LATEST,
+  LATEST_MAJORS
+}
+
+export class MigrationSettings {
+  packages: string[]
+  source: Record<string, string>
+  target: Record<string, string>
+  dryRun: boolean
+  migrationMode: MigrationMode
+  repositoryFieldNewValue?: string
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/joebowbeer/regsync/pullRequest?prid=9263162)
```suggestion
  repositoryFieldNewValue?: string;
```